riak_repl/tests/conftest.py

```diff
@@ -16,5 +16,6 @@ def dd_environment():
     with docker_run(
         compose_file=compose_file,
         conditions=[CheckEndpoints(URL, wait=5), CheckDockerLogs(compose_file, log_patterns, matches='all', wait=5)],
+        sleep=240,
```

Comment:
That's unfortunate that you need to add this. It will make the tests unnecessary slow, can we check a log instead?

Reply:
I believe I have successfully adjusted the log pattern so that sleep is no longer needed.